csv/config5.csv

```diff
@@ -1,8 +1,10 @@
 XLEN     COMPILER    EXTENSIONS  ABI    N_PROC    FNAME
 32      gcc         gc          ilp32   1         simple_add
-32      gcc         gc          ilp32   1         printf
-32      gcc         gc          ilp32   2         func_call
-32      gcc         iaf         ilp32   3         simple_add
+32      gcc         gc          ilp32   32         simple_add
+#32      gcc         gc          ilp32   3         simple_add
+#32      gcc         gc          ilp32   1         printf
+#32      gcc         gc          ilp32   2         func_call
+#32      gcc         iaf         ilp32   3         simple_add
 #32     gcc         ia          ilp32
```

Comment:
Please remove unused rows rather than just commenting them out.
If you want to keep stuff around for reference, then you can put it on a branch.